go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/rancher/hosted-providers-e2e
 
-go 1.19
+go 1.20
 
 require (
 	github.com/Masterminds/semver/v3 v3.2.1
-	github.com/epinio/epinio v1.10.0
-	github.com/onsi/ginkgo/v2 v2.13.1
-	github.com/onsi/gomega v1.30.0
+	github.com/epinio/epinio v1.11.0
+	github.com/onsi/ginkgo/v2 v2.15.0
+	github.com/onsi/gomega v1.31.1
 	github.com/pkg/errors v0.9.1
-	github.com/rancher/rancher v0.0.0-20231121113554-764f891cfa95
-	k8s.io/apimachinery v0.27.6
-	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	github.com/rancher/rancher v0.0.0-20240118221102-fa9e804ff2d7
```

Comment:
lgtm.
Is this commit from v2.8.1 ? We should take latest commit from v2.8 branch instead